tests/secrets/resources/cfn/test-multiline-secrets.yml

```diff
@@ -0,0 +1,15 @@
+spec:
```

Comment:
@marynaKK perhaps move to a separate dir to verify you don't affect other tests?